octoprint_mrbeam/analytics/analytics_handler.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, plugin):
         # Job-specific data
         self._current_job_id = None
         self._current_job_time_estimation = -1
```

Comment:
Please rename all the `self._current_job_time_estimation` into `self._current_job_time_estimation_v1`

Reply:
and please add a comment to explain which is which (frontend vs backend maybe)